README.md

```diff
@@ -110,6 +110,28 @@ If you're using Botfront in a production environment, consider adding your own d
 **Important**
 If you are using the provided MongoDB deployment, `mongodb.mongodbHost` must follow the following pattern: `<release-name>-mongodb-service.<namespace>`
 
+
+### Enable and configure redis
+
+Redis is a data structure store, used by rasa as a lockstore. It should be enabled when working with multiple rasa instances. As it ensure that
+incoming messages are processed in the right order.
+
+| Parameter                            | Description                                                   | Default                             |
+|--------------------------------------|---------------------------------------------------------------|-------------------------------------|
+| `redis.enabled`                      | Set to `true` to enable redis                                 | `false`                             |
+| `redis.cluster.enabled`              | Do not change this value                                      | `false`                             |
+| `redis.cluster.slaveCount`           | Do not change this value                                      | 0                                   |
```

Comment:
Why exposing them if they cannot be changed?

Reply:
Or you can expose it but leave the doc in the values.yaml

Reply:
I removed those from the docs and added the warning in the values.yaml